package.json

```diff
@@ -7,7 +7,7 @@
     "@storybook/addon-info": "3",
     "@storybook/addon-knobs": "3",
     "@storybook/react": "^3.0.0",
-    "axios": "0",
+    "axios": ">=0.18.1",
```

Comment:
what does `>=` do other than `^` does? Don't you need to specify a range when using `>=` ?

Reply:
oh hang on something to do with only using stable versions?

Reply:
Just using the suggest range from GH: https://github.com/comicrelief/storybook/network/alerts

Reply:
hm ok can't access that. But i guess it's fine.